internal/query/projection/user_personal_access_token.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	PersonalAccessTokenProjectionTable = "projections.personal_access_tokens3"
+	PersonalAccessTokenProjectionTable = "projections.personal_access_tokens4"
```

Comment:
I'd prevent creating a new iteration as it might take quite some time with larger systems. The `owner_removed` column can be cleaned up when going relational and the indeces will be created when not yet existing.